opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -87,93 +88,121 @@ message InstrumentationLibraryMetrics {
   string description = 2;
 
-  // unit in which the metric value is reported. Follows the format
+  // Unit in which the metric value is reported. Follows the format
   // described by http://unitsofmeasure.org/ucum.html.
   string unit = 3;
 
-  // Type is the type of values a metric has.
-  enum Type {
-    // INVALID_TYPE is the default Type, it MUST not be used.
-    INVALID_TYPE = 0;
-
-    // INT64 values are signed 64-bit integers.
-    //
-    // A Metric of this Type MUST store its values as Int64DataPoint.
-    INT64 = 1;
-
-    // MONOTONIC_INT64 values are monotonically increasing signed 64-bit
-    // integers.
-    //
-    // A Metric of this Type MUST store its values as Int64DataPoint.
-    MONOTONIC_INT64 = 2;
-
-    // DOUBLE values are double-precision floating-point numbers.
-    //
-    // A Metric of this Type MUST store its values as DoubleDataPoint.
-    DOUBLE = 3;
-
-    // MONOTONIC_DOUBLE values are monotonically increasing double-precision
-    // floating-point numbers.
-    //
-    // A Metric of this Type MUST store its values as DoubleDataPoint.
-    MONOTONIC_DOUBLE = 4;
-
-    // Histogram measurement.
-    // Corresponding values are stored in HistogramDataPoint.
-    HISTOGRAM = 5;
-
-    // Summary value. Some frameworks implemented Histograms as a summary of observations
-    // (usually things like request durations and response sizes). While it
-    // also provides a total count of observations and a sum of all observed
-    // values, it calculates configurable percentiles over a sliding time
-    // window.
-    // Corresponding values are stored in SummaryDataPoint.
-    SUMMARY = 6;
+  // ValueType is the type of values a metric has.  ValueType determines
+  // which field of the DataPoint will be used for Metrics with this
+  // descriptor.
+  enum ValueType {
+    // INVALID_TYPE is the default ValueType, it MUST not be used.
+    INVALID_VALUE_TYPE = 0;
+
+    // Note that int64 ValueTypes use the even numbers and double
+    // ValueTypes use the odd numbers.
+
+    // ScalarInt64 implies that Metric.Points MUST use DataPoint.value_int64.
+    SCALAR_INT64     = 2;
+    // ScalarDouble implies that Metric.Points MUST use DataPoint.value_double.
+    SCALAR_DOUBLE    = 3;
+
+    // HistogramInt64 implies that Metric.Points MUST use
+    // DataPoint.histogram with int64 boundaries.
+    HISTOGRAM_INT64  = 4;
+    // HistogramDouble implies that Metric.Points MUST use
+    // DataPoint.histogram with floating point boundaries.
+    HISTOGRAM_DOUBLE = 5;
+
+    // SummaryInt64 implies that Metric.Points MUST use DataPoint.summary
+    // with int64 min/max/sum/last/count values.
+    SUMMARY_INT64    = 6;
+    // SummaryDouble implies that Metric.Points MUST use
+    // DataPoint.summary with double min/max/sum/last/count values.
+    SUMMARY_DOUBLE   = 7;
+
+    // RawInt64 implies that Metric.Points MUST use DataPoint.raw with
+    // int64 values.
+    RAW_INT64        = 8;
```

Comment:
As discussed in the metrics SIG, raw values can be sent in two cases, which is confusing (or maybe I got confused):
1. As exemplars, together with other value type.
2. Only raw metrics, for cases where individual measurements are sent.

At least that's what I understand, if that is not the case please clarify.

Probably would be better to separate the entries in the DataPoint - raw and exemplars and use the same message for both?

Reply:
> Probably would be better

Agreed.

Reply:
Done. It now states that raw_values and exemplars should not both be set.